neuromation/api/blob_storage.py

```diff
@@ -217,6 +218,7 @@ def __init__(self, core: _Core, config: Config) -> None:
     async def glob_blobs(self, bucket_name: str, pattern: str) -> List[BlobListing]:
         pattern = pattern.lstrip("/")
         parts = pattern.split("/")
```

Comment:
Humm, the pattern can contain something like `[-/]`.

Reply:
Then it would be an invalid URL.

Reply:
It is not a URL, it is a pattern. `foo[-/]bar` matches two paths: `foo-bar` and `foo/bar`.

Reply:
Is that a valid glob case?